scripts/gen_vpc_ip_limits.go

```diff
@@ -200,18 +206,7 @@ func addManualLimits(limitMap map[string]awsutils.InstanceTypeLimits) map[string
 		"u-9tb1.metal":  {ENILimit: 5, IPv4Limit: 30, HypervisorType: "unknown", IsBareMetal: true},
 		"c5a.metal":     {ENILimit: 15, IPv4Limit: 50, HypervisorType: "unknown", IsBareMetal: true},
 		"c5ad.metal":    {ENILimit: 15, IPv4Limit: 50, HypervisorType: "unknown", IsBareMetal: true},
-		"p4d.24xlarge":  {ENILimit: 15, IPv4Limit: 50, HypervisorType: "unknown", IsBareMetal: false},
 		"p4de.24xlarge": {ENILimit: 15, IPv4Limit: 50, HypervisorType: "unknown", IsBareMetal: false},
-		"dl1.24xlarge":  {ENILimit: 15, IPv4Limit: 50, HypervisorType: "unknown", IsBareMetal: false},
-		"c6g.xlarge":    {ENILimit: 4, IPv4Limit: 15, HypervisorType: "nitro", IsBareMetal: false},
-		"c7g.12xlarge":  {ENILimit: 8, IPv4Limit: 30, HypervisorType: "nitro", IsBareMetal: false},
-		"c7g.16xlarge":  {ENILimit: 15, IPv4Limit: 50, HypervisorType: "nitro", IsBareMetal: false},
-		"c7g.2xlarge":   {ENILimit: 4, IPv4Limit: 15, HypervisorType: "nitro", IsBareMetal: false},
-		"c7g.4xlarge":   {ENILimit: 8, IPv4Limit: 30, HypervisorType: "nitro", IsBareMetal: false},
-		"c7g.8xlarge":   {ENILimit: 8, IPv4Limit: 30, HypervisorType: "nitro", IsBareMetal: false},
-		"c7g.large":     {ENILimit: 3, IPv4Limit: 10, HypervisorType: "nitro", IsBareMetal: false},
-		"c7g.medium":    {ENILimit: 2, IPv4Limit: 4, HypervisorType: "nitro", IsBareMetal: false},
-		"c7g.xlarge":    {ENILimit: 4, IPv4Limit: 15, HypervisorType: "nitro", IsBareMetal: false},
```

Comment:
I removed these based on the scripts output (with the above modifications):
```
Delete "c7g.xlarge": {4 15 nitro false} is already correct in the API
Adding "p4de.24xlarge": {15 50 unknown false} since it is missing from the API
Adding "c7g.metal": {15 50 unknown true} since it is missing from the API
Delete "c7g.16xlarge": {15 50 nitro false} is already correct in the API
Adding "c5ad.metal": {15 50 unknown true} since it is missing from the API
Delete "c7g.medium": {2 4 nitro false} is already correct in the API
Adding "hs1.8xlarge": {8 30 unknown false} since it is missing from the API
Adding "u-9tb1.metal": {5 30 unknown true} since it is missing from the API
Adding "c5a.metal": {15 50 unknown true} since it is missing from the API
Adding "u-6tb1.metal": {5 30 unknown true} since it is missing from the API
Replacing API value {15 50 nitro false} with override {15 50 unknown false} for "p4d.24xlarge"
Delete "c7g.4xlarge": {8 30 nitro false} is already correct in the API
Delete "c7g.8xlarge": {8 30 nitro false} is already correct in the API
Delete "c7g.large": {3 10 nitro false} is already correct in the API
Adding "cr1.8xlarge": {8 30 unknown false} since it is missing from the API
Adding "u-12tb1.metal": {5 30 unknown true} since it is missing from the API
Adding "u-18tb1.metal": {15 50 unknown true} since it is missing from the API
Delete "c7g.12xlarge": {8 30 nitro false} is already correct in the API
Delete "c7g.2xlarge": {4 15 nitro false} is already correct in the API
Adding "u-24tb1.metal": {15 50 unknown true} since it is missing from the API
Replacing API value {15 50 nitro false} with override {15 50 unknown false} for "dl1.24xlarge"
Delete "c6g.xlarge": {4 15 nitro false} is already correct in the API
```